docs/build_docs

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Build API docs then build docusaurus docs.
+# Currently used in our netlify pipeline.
+# Note: Will be modified to build API docs on latest tagged release.
+
+echo "Installing dev dependencies"
```

Comment:
Wanting to check this too : Does the behavior here change in light of the Arch Review today?

Reply:
Yep! That work is in the next ticket [DX-150](https://greatexpectations.atlassian.net/browse/DX-150). There is sort of a chicken / egg problem here so I was planning to get these two documents released in the next PR under the 149 ticket and then follow up with updating / codifying the release process in 150. This script is not yet used in our main Netlify build.